src/Spaces/box.jl

```diff
@@ -25,7 +25,7 @@ function Box(low::Number, high::Number, shape::Union{Tuple, Array{Int64, 1}}, dt
         @warn "dtype was autodetected as $(dtype). Please provide explicit data type."
     end
     if dtype <: Integer
-        low = floor(dtype, low)
+        low = ceil(dtype, low)
```

Comment:
I think we should display a warning here.

Reply:
**Check**